levels/menu/script.c

```diff
@@ -42,6 +44,7 @@ const LevelScript level_main_menu_entry_file_select[] = {
     END_AREA(),
 
     FREE_LEVEL_POOL(),
+#ifndef SKIP_FILE_SELECT
```

Comment:
I'm a little iffy on this define because it's pretty much functionally identical to `TEST_LEVEL`, just without the ability to change the starting level. If we are to keep it though, I think you should be able to just stub out all but the last two lines in this script, since the rest of it probably isn't necessary for anything.

Reply:
addressed